3-node-cluster/ansible/playbooks/setup-pgpool.yml

```diff
@@ -41,8 +41,8 @@
         - { regexp: "#backend_weight1 = 1", replace: "backend_weight1 = 0.5" } # Set the weight of the standby1 PostgreSQL node to 1 for load balancing.
         - { regexp: "#backend_data_directory0 = '/data'", replace: "backend_data_directory0 = '{{ PG_CIRRUS_INSTALLATION_DIRECTORY }}'" } # Data directory path for primary PostgreSQL server
         - { regexp: "#backend_data_directory1 = '/data1'", replace: "backend_data_directory1 = '{{ PG_CIRRUS_INSTALLATION_DIRECTORY }}'" } # Data directory path for standby1 PostgreSQL server
-        - { regexp: "#failover_command = ''", replace: "failover_command = '{{ ansible_env.PWD }}/../../pgpool-internals/failover.sh %d %h %p %D %m %H %M %P %r %R %N %S {{ PG_VERSION }} {{ PG_PORT }}'" } # Parameters for failover_command are set
-        - { regexp: "#follow_primary_command = ''", replace: "follow_primary_command = 'sudo {{ ansible_env.PWD }}/../../pgpool-internals/follow_primary.sh %d %h %p %D %m %H %M %P %r %R %N %S {{ PG_PORT }} '" } # Parameters for follow_primary_command are set
+        - { regexp: "#failover_command = ''", replace: "failover_command = 'su - postgres -c \"u{{ ansible_env.PWD }}/../../pgpool-internals/failover.sh %d %h %p %D %m %H %M %P %r %R %N %S {{ PG_VERSION }} {{ PG_PORT }}\"'" } # Parameters for failover_command are set
```

Comment:
@salmanahmed-postgresql Could you please resider this `su - postgres -c \"u{`
I dont think there should be u before {

Reply:
Thanks for pointing out, you are right the u is not needed. To undo in the text editor that I use we have to press ctrl + u. This must have been added by mistake. I'll remove it.